vault/sources/Vault.move

```diff
@@ -341,75 +380,86 @@ module titusvaults::Vault {
     }
 
     // strick price
-    public (friend) fun setStrikePrice(_host: &signer, round_id: u64, new_strike_price: u64) acquires RoundState {
+    public (friend) fun setStrikePrice(_host: &signer, round_id: u64, new_strike_price: u64) acquires VaultMap {
         let host_addr = address_of(_host);
         assert!(host_addr == @titusvaults, E_NOT_AUTHORIZED);
 
-        let state = borrow_global_mut<RoundState>(@titusvaults);
+        let vault_map = borrow_global_mut<VaultMap>(@titusvaults);
+
+        // first round state
+        let first_round_state = vector::borrow_mut(&mut vault_map.rounds, 0); 
```

Comment:
we need to set the strike price on round ID whose ID has been passed to this function as parameter and not the first round ID.